src/main/java/seedu/module/logic/parser/ParserUtil.java

```diff
@@ -82,17 +83,20 @@ public static Description parseDescription(String description) throws ParseExcep
     }
 
     /**
-     * Parses a {@code String module} into an {@code Module}.
-     * Leading and trailing whitespaces will be trimmed.
+     * Parses a {@code String module} into an {@code Module} that is supported.
+     * Leading and trailing whitespaces are trimmed.
      *
      * @throws ParseException if the given {@code module} is invalid.
      */
     public static Module parseModule(String module) throws ParseException {
         requireNonNull(module);
         String trimmedModule = module.trim();
```

Comment:
im not sure if trimming is necessary here as in ModuleBookParser's parse command, userInput.trim() is already called with the matcher. Good to double check there haha